### ISSUE TITLE
Allow to pass extra options over to civicrm-dcoker-install script

### ISSUE THREAD

**Issue by frlan** [civicrm/civicrm-docker#12]
If got an scenario, were I'd like to set the -f / -k flag for cv to override an existing configuration on purpose. I could imagine that in future it could be usful to have an free text field here to just pass whatever needed

**Comment by michaelmcandrew** [comment]
This sounds like you are rolling your own installation command. Given that the installation script is just a single comman, at this point, I would say that you should just call the installation command directly, i.e. something like

```shell
docker compose exec -u www-data cv core:install # whatever you want here
```

**Comment by frlan** [comment]
@michaelmcandrew I disagree on the point to always build the own container for such a small change. The diff for the script is minimalistic (IMHO) and would help a lot to make it more portable (read: accessible/useable for people from community)

**Comment by michaelmcandrew** [comment]
@frlan - I'm not suggesting that you build a new container - just run whatever install command you want on the same container

**Comment by michaelmcandrew** [comment]
Closing for now but feel free to reopen if you want to continue the discussion.
